base/all/DESCRIPTION

```diff
@@ -61,4 +61,4 @@ Copyright: Authors
 LazyLoad: yes
 LazyData: FALSE
 Encoding: UTF-8
-RoxygenNote: 7.0.2
+RoxygenNote: 7.1.2
```

Comment:
built under wrong version of roxygen, need to revert

Reply:
@mdietze @helge22a 7.1.2 is the correct Roxygen version now and is what I see [in this file on the develop branch](https://github.com/PecanProject/pecan/blob/develop/base/all/DESCRIPTION#L80), so I'm puzzled why this change shows in the diff at all

Reply:
Thanks @infotroph. @helge22a let's ignore the warnings and focus on the requested changes and the merge conflicts